ntoskrnl/lpc/create.c

```diff
@@ -92,6 +91,8 @@ LpcpCreatePort(OUT PHANDLE PortHandle,
     if (CapturedObjectName.Length == 0)
         CapturedObjectName.Buffer = NULL;
 
+    LPCTRACE(LPC_CREATE_DEBUG, "Name: %wZ\n", &CapturedObjectName);
```

Comment:
`CapturedObjectName.Buffer` looks like it would still be an untrusted pointer here